inc/class-cachify.php

```diff
@@ -863,70 +854,65 @@ public static function touch_comment( $new_status, $old_status, $comment ) {
 	}
 
 	/**
-	 * Generate publish hook for custom post types
+	 * Removes the post type cache if saved or updated
 	 *
-	 * @since   2.1.7  Make the function public
-	 * @since   2.0.3
+	 * @since
+	 * @change
 	 *
-	 * @return  void
+	 * @param	integer	$id	Post ID
 	 */
-	public static function register_publish_hooks() {
-		/* Available post types */
-		$post_types = get_post_types(
-			array(
-				'public' => true,
-			)
-		);
 
-		/* Empty data? */
-		if ( empty( $post_types ) ) {
-			return;
-		}
+	public static function save_update_trash_post( $id ) {
```

Comment:
Renaming a public function without prior deprecrecation and not even a change note (e.g. third `@since` tag) is not really kind.

So we should either keep and deprecate the old method and introduce a new one or raise the project version to 3.0 (prior deprecation would be nice though)

Reply:
resolved in 7bd42ba7e492bae695062a53f632f91bc0fe2ab1